repos/system_upgrade/common/libraries/config/version.py

```diff
@@ -14,7 +14,7 @@
 _SUPPORTED_VERSIONS = {
     # Note: 'rhel-alt' is detected when on 'rhel' with kernel 4.x
     '7': {'rhel': ['7.9'], 'rhel-alt': [], 'rhel-saphana': ['7.9']},
-    '8': {'rhel': ['8.6', '8.8'], 'rhel-saphana': ['8.6']},
+    '8': {'rhel': ['8.6', '8.8'], 'rhel-saphana': ['8.6', '8.8']},
```

Comment:
:heart:  exactly. I am sorry I missed that. Hopefully it didn't make you a headache guys.